tests/v2/validation/provisioning/permutations/permutations.go

```diff
@@ -61,7 +61,7 @@ func RunTestPermutations(s *suite.Suite, testNamePrefix string, client *rancher.
 						clusterObject, err := provisioning.CreateProvisioningCluster(client, *nodeProvider, testClusterConfig, hostnameTruncation)
 
 						require.NoError(s.T(), err)
-						provisioning.VerifyCluster(s.T(), client, clusterObject)
+						provisioning.VerifyCluster(s.T(), client, clusterObject, clustersConfig)
```

Comment:
Similar to above, I don't think `clustersConfig` should be passed as a parameter.